src/components/Page/LandingPageWidgets.vue

```diff
@@ -1,6 +1,16 @@
 <template>
 	<div class="landing-page-widgets">
-		<MembersWidget v-if="!isPublic" />
+		<div class="first-row-widgets">
+			<MembersWidget v-if="!isPublic" :current-collective="currentCollective" />
+			<NcButton v-if="'contacts' in OC.appswebroots" :href="teamUrl">
```

Comment:
Nitpicking, but maybe it makes sense to move condition `'contacts' in OC.appswebroots` into a computed to cache it?